ballista/rust/client/src/context.rs

```diff
@@ -245,7 +245,7 @@ impl BallistaContext {
             let state = self.state.lock().unwrap();
             for (name, plan) in &state.tables {
                 let plan = ctx.optimize(plan)?;
-                let execution_plan = ctx.create_physical_plan(&plan)?;
+                let execution_plan = ctx.create_physical_plan(&plan).await?;
                 ctx.register_table(
                     TableReference::Bare { table: name },
                     Arc::new(DfTableAdapter::new(plan, execution_plan)),
```

Comment:
I feel `BallistaContext.sql` should not become async. Opening #1016 to track that.